docs/api-output.md

```diff
@@ -93,13 +93,16 @@ Returns **[Promise][5]&lt;[Buffer][8]>** when no callback is provided
 Include all metadata (EXIF, XMP, IPTC) from the input image in the output image.
 This will also convert to and add a web-friendly sRGB ICC profile.
 
+An optional output ICC profile can be specified to replace the default sRGB ICC profile.
+
 The default behaviour, when `withMetadata` is not used, is to convert to the device-independent
 sRGB colour space and strip all metadata, including the removal of any ICC profile.
 
 ### Parameters
 
--   `options` **[Object][6]?** 
+-   `options` **[Object][6]?**
     -   `options.orientation` **[number][9]?** value between 1 and 8, used to update the EXIF `Orientation` tag.
+    -   `options.icc` **[string][2]?** path to an ICC profile file
```

Comment:
This content differs from the JSDoc. Was `npm run docs-build` used to update these?